### DISCUSSION TITLE
Install and upgrade via winget

### DISCUSSION THREAD

**Discussion by jos3s** [JanDeDobbeleer/oh-my-posh#563] [Ideas]
Was it possible that Oh my posh was available for installation via the [winget](https://github.com/microsoft/winget-pkgs)?

**Comment by JanDeDobbeleer** [comment]
Not yet. As soon as Winget supports auto update of the spec files, I'll add it. Today it requires a PR to their repo for every version increment and I'm not going to do that without an automated process.

**Reply by jos3s** [reply]
Yes, I understand I hope this feature is on the winget's priority list

**Reply by JanDeDobbeleer** [reply]
@jos3s it is. Was supposed to be for March. I actually have a working template for winget so once that's available, the work is minimal.

**Reply by JanDeDobbeleer** [reply]
@jos3s see [this](https://github.com/microsoft/winget-pkgs/issues/8900) issue for reference/follow-up

**Reply by jos3s** [reply]
@JanDeDobbeleer They made the change to the new format of the Manifesto, now the OMP will be made available by the winget? 

[Issue Winget](https://github.com/microsoft/winget-pkg/issues/8900)

**Reply by JanDeDobbeleer** [reply]
@jos3s as soon as they release the new CLI with auto-pr functionality.

**Reply by floh96** [reply]
@JanDeDobbeleer the cli for creating a pr was initially [released](https://github.com/microsoft/winget-create/releases/tag/v0.1.0.1-preview) 15 days ago. The [docs](https://github.com/microsoft/winget-create#using-windows-package-manager-manifest-creator-in-a-cicd-pipeline) for use in ci.

**Reply by JanDeDobbeleer** [reply]
@jos3s @floh96 blocked by https://github.com/microsoft/winget-create/issues/38

**Reply by jos3s** [reply]
@JanDeDobbeleer any updates on this?

**Reply by JanDeDobbeleer** [reply]
@jos3s it's available. See the [docs](https://ohmyposh.dev/docs/windows).
